src/management/OrganizationsManager.js

```diff
@@ -0,0 +1,816 @@
+ *   console.log(invites);
+ * });
+ *
+ * @param   {String}    [organization_id]   Organization ID
+ * @param   {Function}  [cb]                Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+OrganizationsManager.prototype.getInvites = function(params, callback) {
+  return this.invitations.getAll(params, callback);
+};
+
+/**
+ * Get an Invitation in a Organization
+ *
+ * @method    getInvite
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params = {id : 'ORGANIZATION_ID', invitation_id: 'INVITATION_ID'}
+ * @example <caption>
+ *   This methods takes the organization ID and user ID and returns the invitation
+ * </caption>
+ *
+ * management.organizations.getInvite( {id : 'ORGANIZATION_ID', invitation_id: 'INVITATION_ID'}, function (err, invite) {
+ *   console.log(invite);
+ * });
+ *
+ * @param   {String}    [organization_id]   Organization ID
+ * @param   {Function}  [cb]                Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+OrganizationsManager.prototype.getInvite = function(params, callback) {
+  return this.invitations.get(params, callback);
+};
+
+/**
+ * Create an invitation in an organization
+ *
+ * @method    createInvite
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params =  { id :'ORGANIZATION_ID'};
+ * var data = {}
+ *
+ * management.organizations.createInvite(params, data, function (err) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ * });
+ *
+ * @param   {String}    params.id             ID of the Organization.
+ * @param   {Array}     data                  Invitation data
+ * @param   {Function}  [cb]                  Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+OrganizationsManager.prototype.createInvite = function(params, data, cb) {
+  data = data || [];
+  params = params || {};
+
+  if (!params.id) {
+    throw new ArgumentError('The organization ID passed in params cannot be null or undefined');
+  }
+  if (typeof params.id !== 'string') {
+    throw new ArgumentError('The organization ID has to be a string');
+  }
+
+  if (cb && cb instanceof Function) {
+    return this.invitations.create(params, data, cb);
+  }
+
+  return this.invitations.create(params, data);
+};
+
+/**
+ * Delete an invitation from an organization
+ *
+ * @method    deleteInvitation
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params =  { id :'ORGANIZATION_ID', invitation_id: 'INVITATION_ID };
+ *
+ * management.organizations.deleteInvitation(params, function (err) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ * });
+ *
+ * @param   {String}    params.id             ID of the Organization.
+ * @param   {String}    params.invitation_id  Invitation ID
+ * @param   {Function}  [cb]                  Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+OrganizationsManager.prototype.deleteInvite = function(params, cb) {
```

Comment:
IMO `deleteInvitation` is a better wording. (applies to all other invitation related endpoint above)